src/RadioButton/index.js

```diff
@@ -0,0 +1,210 @@
+import React, {forwardRef, useCallback, useEffect, useRef} from 'react';
+import styled, {css} from 'styled-components';
+import PropTypes from 'prop-types';
+import {
+	mergeRefs,
+	pxToEm,
+	pxToRem,
+	alpha,
+	contrast,
+	getDimmedTextColor,
+} from '../utils';
+import {fillParent} from '../mixins';
+import Icon from '../Icon';
+import {useTabIndexContext} from '../TabIndexContext';
+
+const EXTEND_TAP_AREA_SPACING = pxToRem(9);
+
+const noop = () => {};
+
+const Wrapper = styled.span`
+	position: relative;
+	display: inline-flex;
+`;
+
+const Input = styled.input`
+	position: absolute;
+	top: -${EXTEND_TAP_AREA_SPACING};
+	left: -${EXTEND_TAP_AREA_SPACING};
+	width: calc(100% + ${EXTEND_TAP_AREA_SPACING} * 2);
+	height: calc(100% + ${EXTEND_TAP_AREA_SPACING} * 2);
+	opacity: 0;
+`;
+
+const MockBox = styled.span.withConfig({
+	shouldForwardProp: prop => !['isChecked', 'scale'].includes(prop),
+})`
+	position: relative;
+	pointer-events: none;
+
+	display: flex;
+	align-items: center;
+	justify-content: center;
+
+	width: ${pxToEm(16)};
+	height: ${pxToEm(16)};
+
+	font-size: ${p => (p.scale === 'auto' ? '1em' : pxToRem(20 * p.scale))};
+
+	color: ${p => (p.isChecked ? p.theme.links : p.theme.background)};
+	background: ${p => contrast(p.theme.links)};
+	border: 1px solid
+		${p =>
+			alpha(
+				p.isChecked ? p.theme.links : p.theme.text,
+				p.isChecked ? 1 : 0.5
+			)};
+	border-radius: ${pxToRem(16)};
+
+	input[disabled] + & {
```

Comment:
Not sure whether the disabled `RadioButtons` should maybe also have an outline. I've left it for now in order to be more consistent with the `Checkbox` UI.

`Checkbox` in comparison:
<img width="1111" alt="Screenshot 2022-01-04 at 11 37 15" src="https://user-images.githubusercontent.com/41924354/148046660-fa7f1b54-87a3-4a32-9e00-1d06585ab0c7.png">

Reply:
Looks fine to me in the screenshot 👍